tfbpshiny/utils/create_distribution_plot.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
```

Comment:
Note the following:

1. this is not a shiny object (not UI or server). It is a function and goes in utils/

2. The name of the file matches the name of the function

3. The input and output types are specified

4. The documentation format is sphinx and more importantly, matches the rest of the repo. In general, you need to make sure that what you're adding matches the format of the repo. This is general advice for anytime you are working in any repo -- match the style of the repo.